lib/actions/app.coffee

```diff
@@ -134,9 +141,24 @@ exports.remove =
 	options: [ commandOptions.yes ]
 	permission: 'user'
 	action: (params, options, done) ->
-		visuals.patterns.remove 'application', options.yes, (callback) ->
-			resin.models.application.remove(params.name).nodeify(callback)
+		async.waterfall([
```

Comment:
Hm, fair enough, not sure why I made that :/ Keep them if you like, we'll get rid of `async` anyway later on.